### DISCUSSION TITLE
python function call failed

### DISCUSSION THREAD

**Discussion by djxhhh** [go-python/cpy3#24] [Usage Discussions]
I found that the call failed when I used the call python function to return nil
![image](https://user-images.githubusercontent.com/55906052/182994937-bca6e037-c1c6-47af-b551-48a7f64a9082.png)
![image](https://user-images.githubusercontent.com/55906052/182995056-a20b5182-2ebb-447d-9c5d-9eeb8d228d91.png)
![image](https://user-images.githubusercontent.com/55906052/182995125-f81f655c-35a9-4cd4-84f1-51df4230b408.png)

**Comment by djxhhh** [comment]
I use the CallObject method to call successfully, but the Call does not work, why？

**Comment by christian-korneck** [comment]
did a PyErr occur and if so what’s the error message?

https://github.com/christian-korneck/python-go/blob/b9bf06c9e65d60d1026dceb47dcdf817d11fc68e/py-bindings/outliers/main.go#L116
